tasks/getHistoricalData.ts

```diff
@@ -289,19 +295,32 @@ task("getHistoricalData", "Retrieves the historical rates")
         if (b >= compoundMainnetStartBlock) {
           try {
             if (cToken && decimals) {
-              let r = await cToken.exchangeRateStored({
-                blockTag: b,
-              });
-              if (decimals > 17) {
-                r = r.div(BigNumber.from(10).pow(decimals - 17));
-              } else if (decimals < 17) {
-                r = r.mul(BigNumber.from(10).pow(17 - decimals));
+              if (taskArgs.rate === "borrow") {
+                let r = await cToken.borrowIndex({
```

Comment:
@ioanaroceanu2000 does this logic take care of interpolation? in other words is it one to one with the compound borrowing rate oracle logic we have in the smart contracts?

Reply:
it was somehow lost in the rebasing process. I replaced it by the interpolation logic now